Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.6
+
+ADD bin/linux/amd64/vcs-webhook /vcs-webhook
```

Comment:
@jw-s pleading ignorance but do we need ca-certificates in this image?

Reply:
@PI-Victor IIRC we need them for vcs-webhook -> coordinator, can't remember though. ATM we use mark grpc client connection as insecure. This will need changed.

Reply:
noticed, definitely needs to be changed, but probably not a priority right now.
i've added the ca-certs regardless. one less thing to worry about.